ledfx_docker/docker-compose-example.yml

```
@@ -0,0 +1,36 @@
+      #- /run/user/1000/pulse/native:/home/ledfx/.config/pulse/pulseaudio.socket # Necessary when running in pulseaudio client mode, to access host's PulseAudio socket
+      #- $HOME/.config/pulse/cookie:/home/ledfx/.config/pulse/cookie:ro # Necessary when running in pulseaudio client mode, to access host's PulseAudio cookie
+
+
+  shairportforledfx:
+    image: mikebrady/shairport-sync:latest
+    container_name: shairportforledfx
+    restart: on-failure:3
+    command: -o pa #Use this only if you do not provide a custom shairport-sync.conf file with pulseaudio backend.
+    depends_on:
+      ledfx:
+       condition: service_healthy #so that the pulseaudio socket is available before being mounted here
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 7

(indentation)